tests/generic_config_updater/templates/create_custom_table_type.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "op": "add",
+        "path": "/ACL_TABLE_TYPE",
+        "value": {
+            "DYNAMIC_ACL_TABLE_TYPE" : {
+            "MATCHES": ["DST_IP","DST_IPV6","ETHER_TYPE","IN_PORTS","L4_DST_PORT","L4_DST_PORT_RANGE","IP_PROTOCOL","IP_TYPE"],
+            "ACTIONS": ["PACKET_ACTION","COUNTER"],
+            "BIND_POINTS": ["PORT"]
+            }
```

Comment:
Could you reformat the json file?

Reply:
What would you like changed about the formatting here?  There aren't any issues with it immediately clear to me

Reply:
reformatted correctly.